module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1394,18 +1396,36 @@ public function renewMyItems($renewDetails)
      */
     public function getPickupLocations($patron, $holdInfo = null)
     {
+        $limitedServicePoints = false;
+        if (
+            str_contains($this->config['Holds']['limitPickupLocations'] ?? '', 'itemEffectiveLocation')
+            && $holdInfo['item_id'] ?? false
```

Comment:
The check for `$holdInfo['item_id']` should I think disable this feature for title-level holds.  Our institution doesn't use those and I'm not sure if there would be any reasonably equivalent logic.

Reply:
This makes sense to me, though it would be good to test this to be sure (and to add a comment as well). I think I can set up title level holds in my test environment fairly easily, but since our setup only has one pickup location, I can't really test with much nuance.... might be good to solicit other interested partners if we can find any!

Reply:
Agreed -- just put out a call on #vufind-integrations.

Reply:
Also I did add a comment in Folio.ini that it should only affect item-level holds.  If you think it's helpful I can add a note here as well.  It's just a guess on my part, assuming that's what the `$holdInfo['item_id'] ?? false` will accomplish.

Reply:
Thanks! I see you've gotten at least one significant reply, so I'll wait and see how that plays out before reviewing further. If this does go forward in its current form, I don't think it would hurt to add a `// If there's no item ID, it must be a title-level hold, so this check does not apply` comment here (or something like that).

Reply:
Fair enough, I added a version of that comment.

Will wait to see if others can comment on title-level holds in general.

Reply:
Sounds good! Let me know if you need me to do any testing on my end -- as noted, my ability to do so is somewhat limited, but I'm willing if it turns out to be better than nothing. :-)